### PR TITLE
ddl: Remove the EncodeRecordKey call and add more log message

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

File: ddl/reorg.go [hunk 1]
```diff
@@ -130,12 +130,15 @@ func (d *ddl) runReorgJob(f func() error) error {
...
 	case err := <-d.reorgDoneCh:
+		log.Info("[ddl] run reorg job done")
 		d.reorgDoneCh = nil
 		return errors.Trace(err)
 	case <-d.quitCh:
+		log.Info("[ddl] run reorg job ddl quit")
 		// we return errWaitReorgTimeout here too, so that outer loop will break.
 		return errWaitReorgTimeout
 	case <-time.After(waitTimeout):
+		log.Infof("[ddl] run reorg job wait timeout :%v", waitTimeout)
```
Comment (ngaut) [inline ddl/reorg.go]: We'd better also log job information.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

File: ddl/reorg.go (hunk 1 above)
Comment (zimulala) [inline ddl/reorg.go]: There is no job information. And We know the job information from other information.

**Pull request merged**